tests/test_ui_extra.py

```
@@ -87,3 +87,60 @@ def capture_on(evt, handler):
+            async def pick_mock(*args, **kwargs):
+                return ["/mock/schema/path"]
+            mock_picker.side_effect = pick_mock
+
+            mock_schema_manager.schema_filepath = "/mock/schema/path"
+            mock_schema_manager._load_schema = MagicMock()
+
+            try:
+                mock_schema_manager._load_schema()
+                ui_inst.refresh_tree_and_editor()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)